jp/components/grid_multi_column_headers.md

```diff
@@ -44,11 +44,11 @@ For achieving `n-th` level of nested headers, the declaration above should be fo
 </igx-grid>
 ```
 
-Every `igx-column-group` supports `moving`, `pinning` and `hidding`.
+各 `igx-column-group` は、`moving`、`pinning`、`hidding` をサポートします。
```

Comment:
Actually the feature name should be `hiding`.  If it's not, that's a product bug.